src/languages/php.js

```diff
@@ -379,12 +434,25 @@ export default function(hljs) {
       },
       VARIABLE,
       FUNCTION_INVOKE,
+      LEFT_AND_RIGHT_SIDE_OF_DOUBLE_COLON,
+      {
+        match: [
+          /const/,
+          /\s/,
+          IDENT_RE,
+          /\s*=/,
```

Comment:
Should be a look-ahead, I'm not sure why we need to consume it?

Reply:
Hmm, I don't know enough about the highlight js to know what's the difference.
1. Why can't we consume it? 
2. Why is look-ahead, is a better solution?

Reply:
Because we don't just consume things arbitrarily... we aren't highlighting it here, we only need to guarantee it's presence, not consume it.  In general we don't consume things we don't need to.

Later someone may want to add operator matching where we highlight all `=` and consuming it here would make that impossible...  it should be left int he input stream so another rule might match it later.